src/contexts/field-focus.js

```diff
@@ -55,7 +55,6 @@ const fieldFocusReducer = (state, action) => {
         arg: {on: action.arg.on.getRelatedBy()},
       });
     case 'focusDown':
-      const { field, of } = action.arg;
```

Comment:
> There seem to be some issues with "focus" now

This was the problem. `field` is declared at the top of the function but it gets shadowed by this `field` argument because I guess the block scope is not per `case` statement but within the entire `switch` statement.

Reply:
This fixes the focus in schema-menu, it now will focus correctly and "stay" focused on a click.
I really like how you can focus on an attribute then go "up" and then "down" to see all the attributes for that entity.

I thought there were some new issues just now when I was testing focusing on `drupal_internal__uid` because some rows would collapse entirely, but I think that's because the user id is "0" (anonymous) and not related to this change.